x/ibc/02-client/types/state.go

```diff
@@ -4,7 +4,7 @@ package types
 // Any actor holding the Stage can access on and modify that client information.
 type State struct {
 	// Client ID
-	id string
+	Id string `json:"id" yaml:"id"`
```

Comment:
cc: @cwgoes do we want to expose the client state ID though client queries? 

```suggestion
	ID string `json:"id" yaml:"id"`
```

Reply:
Reference https://github.com/cosmos/cosmos-sdk/pull/5275#discussion_r342414413, I'm not sure we should do this. In fact, we don't have to store the ID in value because the key contains it.